### PR TITLE
Updating Readme to add info on namespaced controllers

### PR DESCRIPTION
**Pull request by Xqua**
I banged my head around the issue board to finally understand all of the information I just added ! 

I think the Api namespace is definitely common enough to warrant its own little example ? 
I wished I could have found this here instead of having to look through a lot of issue threads to finally understand this ! 

Especially the model_name part on a namespace controller !

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,6 +111,14 @@ end
...
 end
+
+# If your controller is under a namespace other than root (example: Api::ModelController) then for merit to find your object automatically, you must specify the model class and not forget that your action is of the form namespace/models#action 
+# Here the Post model belong_to :user which has_many :posts so that the relation post.user exists (to: :user) and we can then access the metric of interest (.user.posts.count)
+
+grant_on 'api/posts#create', badge: 'first-post', model_name: 'Post', to: :user do |post|
+    post.user.posts.count >= 1
+end
+
 ```
```
Comment (tute) [inline README.md:122]:
```suggestion
If your controller is under a namespace other than root (example:
`Api::ModelController`) then for merit to find your object automatically you
must specify the model class and not forget that your action is of the form
`namespace/models#action`.

See an example of a `Post` model that belongs to user:

```ruby
grant_on 'api/posts#create', badge: 'first-post', model_name: 'Post', to: :user do |post|
  post.user.posts.count >= 1
end
```
```

**Comment by Xqua** [comment]
Awesome ! Glad to be of help !

**Comment by tute** [comment]
If you want to push the formatting updates yourself, then I can merge keeping you as author. Let's do it that way if you'd like.

**Comment by tute** [comment]
Thank you @Xqua! Merged it in, and README updated.

**Pull request merged**